python/datashaper/datashaper/engine/verbs/utils/unhot_utils.py

```diff
@@ -6,15 +6,21 @@
 from datashaper.engine.verbs.verb_input import VerbInput
 
 
-def unhotOperation(input: VerbInput, columns: List[str], prefix: str):
+def unhot_operation(input: VerbInput, columns: List[str], prefix: str):
```

Comment:
I know this PR is not about this, but was wondering if pandas melt operation would do this without the custom implementation? 

https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.melt.html

Reply:
That's a good thought, I'll add it as an issue